toolboxes/Containerfile.bluefin-cli

```
@@ -0,0 +1,24 @@
+FROM ghcr.io/homebrew/brew
+# From https://github.com/toolbx-images/images/tree/main/ubuntu/22.04
+
+LABEL com.github.containers.toolbox="true" \
+      usage="This image is meant to be used with the toolbox or distrobox command" \
+      summary="The Bluefin Command Line experience" \
+      maintainer="jorge.castro@gmail.com"
+
+COPY ./toolboxes/packages.bluefin-cli /toolbox-packages
+
+RUN apt-get update && \ 
+    apt-get upgrade -y && \
+    DEBIAN_FRONTEND=noninteractive apt-get -y install \
+        $(cat toolbox-packages | xargs) && \
+    rm -rd /var/lib/apt/lists/*
```

Maybe you need to switch user to `root` for this?  The Dockerfile in Homebrew uses a `linuxbrew` user, which mustn't have permissions

```suggestion
USER root

RUN apt-get update && \ 
    apt-get upgrade -y && \
    DEBIAN_FRONTEND=noninteractive apt-get -y install \
        $(cat toolbox-packages | xargs) && \
    rm -rd /var/lib/apt/lists/*
```